builtins/web/blob.cpp

```diff
@@ -444,20 +442,19 @@ bool Blob::append_value(JSContext *cx, HandleObject self, HandleValue val) {
 
     if (line_endings(self) == LineEndings::Native) {
       auto converted = convert_line_endings_to_native(chars);
-      auto src = converted.data();
+      auto *src = converted.data();
       auto len = converted.length();
       return blob->append(src, src + len);
 
-    } else {
-      auto src = chars.ptr.get();
+    }       auto *src = chars.ptr.get();
       auto len = chars.len;
       return blob->append(src, src + len);
```

Comment:
(Alternatively, just fold this into a single line, inlining `src` and `len`)
```suggestion
    }
    auto *src = chars.ptr.get();
    auto len = chars.len;
    return blob->append(src, src + len);
```